### ISSUE TITLE
False warning on yunohost service check

### ISSUE THREAD

**Issue by Josue-T** [YunoHost/package_linter#89]
Hello,
I got this error
```
 ⓘ  Consider adding a change_url script to support changing where the app is installed
 ! Found some inconsistencies in the 'yunohost service add' commands between install, upgrade and restore:
   install : 
      ...None?...
   upgrade : 
      ...None?...
   restore : 
      yunohost service add uwsgi-app@$app --log /var/log/uwsgi/$app/$app.log
```
But this warning don't really make sense as I use the [uwsgi helper](https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/uwsgi/fuwsgi) which already do this for the install / upgrade / remove script.

For me there are 2 possibility:
- Remove the service management of this helper and let's the packager to mange this. I this case think that it would be also good to add a warning if the packager don't remove the service in the remove script.
- Remove this warning in case of a helper do this job for the install/upgrade/remove.

**Comment by Josue-T** [comment]
Any new about this ?

**Comment by alexAubin** [comment]
Hmf yeah I forgot about this issue ... I don't really know how to fix that, clearly that's the limit of a naive static analysis of the script using greps and stuff ...

On the other hand it's also related to the fact that, generally speaking, the backup/restore script ain't really a symetric of install/remove, and by that I mean that in one case we do for example `ynh_add_nginx_config`, but in the backup/restore script we refer directly to the `/etc/nginx/conf.d/$domain.d/$app.conf` file instead of having for example an `ynh_backup/restore_nginx_config`

So same things happen here ... (except we're talking about a custom helper). Naively I'd say "create a ynh_restore_uwsgi helper and call that in the restore script instead of manually calling the correspond 2~3 commands" ...

**Comment by Josue-T** [comment]
Thanks, OK, I'll do this.
